nilearn/_utils/estimator_checks.py

```diff
@@ -103,6 +106,14 @@
     "check_fit_check_is_fitted": (
         "replaced by check_masker_fitted or check_glm_is_fitted"
     ),
+    "check_fit_check_is_fitted": "replaced by check_masker_fitted",
```

Comment:
The same key exists in the above line. Is this line a replacement for that?

Reply:
yeah I fixed that: pre-commit / ruff were complaining about it